src/asn.rs

```diff
@@ -275,6 +277,230 @@ impl fmt::Display for Asn {
 }
 
 
+//------------ SmallAsnSet --------------------------------------------------
+
+/// A relatively small set of ASNs.
```

Comment:
Even with `AsPath` moving to its own module, maybe we should stick a sentence in here somewhere clarifying that this has nothing to do with the concept of `AS_SET`?

Reply:
I actually had that when it was called `AsnSet`, but removed it when renaming to `SmallAsnSet` because I thought it is somewhat clear that it is a different thing?

(Also, obviously, this whole thing severely lacks in documentation.)